### PR TITLE
Fix CRAN check warnings/notes

### PR DESCRIPTION
**Pull request by hetong007**
Issue: https://github.com/dmlc/xgboost/issues/3963

### CODE REVIEW

**Comment by hetong007** [comment]
@khotilov Could you please help to review this PR? Thanks.

File: Makefile [hunk 1]
```diff
@@ -260,7 +260,8 @@ Rpack: clean_all
 	cp ./LICENSE xgboost
 	cat R-package/src/Makevars.in|sed '2s/.*/PKGROOT=./' | sed '3s/.*/ENABLE_STD_THREAD=0/' > xgboost/src/Makevars.in
 	cp xgboost/src/Makevars.in xgboost/src/Makevars.win
-	sed -i -e 's/@OPENMP_CXXFLAGS@/$$\(SHLIB_OPENMP_CFLAGS\)/g' xgboost/src/Makevars.win
+	sed -i -e 's/@OPENMP_CXXFLAGS@/$$\(SHLIB_OPENMP_CXXFLAGS\)/g' xgboost/src/Makevars.win
+	sed -i -e 's/-pthread/$$\(SHLIB_PTHREAD_FLAGS\)/g' xgboost/src/Makevars.win
```
Comment (khotilov) [inline Makefile]: Do these two lines have any effect on Makevars.win? It contains neither `@OPENMP_CXXFLAGS@` nor `-pthread`.

Comment (hetong007) [reply Makefile]: Note when submitted to CRAN we have `cp xgboost/src/Makevars.in xgboost/src/Makevars.win`. Therefore it actually works on the `Makevars.in` file in github.

**Comment by hetong007** [comment]
Just pushed an update to fix a new warning in https://win-builder.r-project.org/PchtWBmCq1mP/00check.log

File: Makefile (hunk 1 above)
Comment (khotilov) [reply Makefile]: Got it. Thanks.

**Pull request merged**